packager/packager.js

```diff
@@ -198,11 +203,15 @@ function statusPageMiddleware(req, res, next) {
 }
 
 function getAppMiddleware(options) {
+  var transformerPath = options.transformer;
+  if (!path.isAbsolute(transformerPath)) {
```

Comment:
This is only available on very recent node (0.12) and io.js versions. What's the supported version for the packager?